### DISCUSSION TITLE
Fabric JS upgrade to v5 [@typescript/fabric]

### DISCUSSION THREAD

**Discussion by r-ashraf** [DefinitelyTyped/DefinitelyTyped#60364] [Issues with a @types package]
Are there any plans to upgrade fabric js types to sync with v5 of fabric js
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/fabric

[a comment by an automated account is not shown]

**Reply by r-ashraf** [reply]
@martin-badin  @mistriel

**Reply by r-ashraf** [reply]
@oklemencic @joewashear007 @mrand01 @NotWoods @bmartinson @RogerioTeixeira @BradleyHill @bmkrol823 @glenngartner @codertx @mike667 @nataliemarleny @buzzyboy

**Comment by yadidl** [comment]
There is any news about this? or some workaround?
